.eslintrc

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
What's the reason for making this a JSON file instead of YAML? I'd prefer YAML tbh mostly because JSON doesn't allow for comments